configuration/db_mongo.rst

```diff
@@ -88,6 +83,35 @@ This works the same as with the normal Firely Server database, except that you:
 
     For MongoDb it is essential to retain the ``.vonk-import-history.json`` file. Please read :ref:`vonk_conformance_history` for details.
 
+.. _mongodb_transactions:
+
+MongoDB Transactions
+--------------------
+
+.. note::
+    When utilizing MongoDb transactions we strongly advise to use MongoDb v4.2 or higher.
+
+In Firely Server versions prior to v4.9 transactions were simulated for development and test purposes. From Firely Server v4.9 and onwards transactions using MongoDb are now fully supported.
+
+With MongoDb transactions, there are a few things to consider:
+
+#. Firely Server only uses transactions in the following cases:
+
+    #. When uploading a transaction bundle.
+    #. When performing a conditional delete that targets more than one resource.
+
+#. MongoDb transactions in Firely Server always use Read Concern `"snapshot"` and Write Concern `"majority"`, regardless of what was specified through the connection string.
+#. Please read the official MongoDb documentation for production considerations when using transactions: `MongoDb manual <https://www.mongodb.com/docs/manual/core/transactions-production-consideration/>`_
+#. Using the correct Read and Write Concern for your MongoDb replica set is very important:
+
+    #. For PSA (Primary-Secondary-Arbiter) replica sets the Write Concern should be `"w=1"`
+    #. For PSS (Primary-Secondary-Secondary) replica sets the Write Concern should be `"w=majority"`
+    #. Choosing the right Read Concern level is dependent on your use case, you should try and test what works best for you.
+    #. The Read and Write Concern can be set through the connection string in Firely Server, for example: `"mongodb+srv://fs-cluster0.mongodb.net/firely?w=majority&readConcernLevel=local"`
```

Comment:
I am not quite getting this part. You say that FS always uses the same configuration, yet you instruct users to specify the read/write concern.

Reply:
This part is indeed not very clear. :-) It is about  how you connect to your replica set and not with transactions. I moved this part up to the `Using MongoDB` section. The R/W Concern for the database/collection can be defined separately from that of a transaction. For transactions in Firely Server the R/W Concern is hard-coded because otherwise it will not work for us. But for accessing the MongoDb replica set, our customers will have their own configuration for R/W concern (or at the least rely on the default level set by MongoDb). I hope this makes it a bit more clearer, please let me know if that is not the case.